pyoptsparse/pyALPSO/meson.build

```diff
@@ -1,13 +1,13 @@
-python_sources = [
-    '__init__.py',
-    'alpso.py',
-    'alpso_ext.py',
-    'pyALPSO.py',
-    'LICENSE'
-]
+# python_sources = [
```

Comment:
Are these comment outs intended?

Reply:
Yeah it doesn't really matter, these files will be gone as part of #431 and this file was never used anyways due to `subdir('pyALPSO')` being commented out. I can revert to keep the git history a bit cleaner